apps/mark-scan/frontend/src/pages/print_page.tsx

```diff
@@ -1,25 +1,38 @@
-import { useContext } from 'react';
-import { PrintPage as MarkFlowPrintPage } from '@votingworks/mark-flow-ui';
+import React, { useContext } from 'react';
 import { assert } from '@votingworks/basics';
-import { useCurrentLanguage } from '@votingworks/ui';
+import { Icons, P, appStrings, useCurrentLanguage } from '@votingworks/ui';
 import { BallotContext } from '../contexts/ballot_context';
 import { printBallot } from '../api';
+import { CenteredCardPageLayout } from '../components/centered_card_page_layout';
 
 export function PrintPage(): JSX.Element | null {
   const { ballotStyleId, precinctId, votes } = useContext(BallotContext);
   const languageCode = useCurrentLanguage();
-  const printBallotMutation = printBallot.useMutation();
+  const printBallotMutate = printBallot.useMutation().mutate;
 
-  function print(): void {
+  React.useEffect(() => {
     assert(ballotStyleId !== undefined);
     assert(precinctId !== undefined);
-    printBallotMutation.mutate({
+
+    printBallotMutate({
       languageCode,
       precinctId,
       ballotStyleId,
       votes,
     });
-  }
 
-  return <MarkFlowPrintPage print={print} />;
```

Comment:
Are we not deleting the `MarkFlowPrintPage` component because it's used by VxMark still?

Reply:
Yeah, initially started to consolidate between `Mark` and `MarkScan`, but realised we'd need to move the `CenteredCardPageLayout` component to mark-flow-ui first, so punted on that for now